docs/geedocs/5.2.1/answer/7160000.html

```diff
@@ -11,6 +11,7 @@
 <div class="container">
 <div class="sidebar1">
   <p class="sidebar-toc"><a href="../answer/7160000.html" class="current-file">Open GEE 5.2.1</a></p>
+  <p class="sidebar-toc"><a href="../../5.2.0/answer/7160000.html">Open GEE 5.2.0</a></p>
```

Comment:
The other pages also need to have the release notes TOC updated

Reply:
Done